apps/spruce/src/components/SpruceForm/FieldTemplates/ObjectFieldTemplates/index.tsx

```diff
@@ -20,15 +21,26 @@ export const ObjectFieldTemplate = ({
 }: ObjectFieldTemplateProps) => {
   const errors = uiSchema["ui:errors"] ?? [];
   const warnings = uiSchema["ui:warnings"] ?? [];
+  const tooltipTitle = uiSchema["ui:tooltipTitle"] ?? null;
   return (
     <fieldset css={uiSchema["ui:fieldSetCSS"]} id={idSchema.$id}>
-      {(uiSchema["ui:title"] || title) && (
-        <TitleField
-          id={`${idSchema.$id}__title`}
-          title={title || uiSchema["ui:title"]}
-          required={required}
-        />
-      )}
+      {(uiSchema["ui:title"] || title) &&
+        (tooltipTitle ? (
+          <TitleContainer>
+            <TitleField
+              id={`${idSchema.$id}__title`}
+              title={title || uiSchema["ui:title"]}
+              required={required}
+            />
+            <InfoSprinkle>{tooltipTitle}</InfoSprinkle>
+          </TitleContainer>
+        ) : (
+          <TitleField
+            id={`${idSchema.$id}__title`}
+            title={title || uiSchema["ui:title"]}
+            required={required}
+          />
+        ))}
```

Comment:
could we simplify to:
```suggestion
        (
        <TitleContainer>
          <TitleField
            id={`${idSchema.$id}__title`}
            title={title || uiSchema["ui:title"]}
            required={required}
          />
          {tooltipTitle && <InfoSprinkle>{tooltipTitle}</InfoSprinkle>}
        </TitleContainer>
      )
```